test/specs/collections/Breadcrumb/BreadcrumbSection-test.js

```diff
@@ -6,46 +6,26 @@ import sandbox from 'test/utils/Sandbox-util'
 describe('BreadcrumbSection', () => {
   common.isConformant(Breadcrumb.Section)
   common.rendersChildren(Breadcrumb.Section)
+  common.propValueOnlyToClassName(Breadcrumb.Section, 'active')
```

Comment:
I believe you meant to use key only instead of value only:

```diff
-common.propValueOnlyToClassName(Breadcrumb.Section, 'active')
+common.propKeyOnlyToClassName(Breadcrumb.Section, 'active')
```

Key only will use the key name (`active`) as the className whereas value only will use the value.  Example, `color='red'` uses value only, since we only the value `red` as the className.


Also, it is helpful to run `npm run test:watch` and use [exclusive tests](https://mochajs.org/#exclusive-tests) to reduce the clutter and identify issues.

Lastly, you can always click `details` on the ci/circle ci check on the PR to see the issue: https://circleci.com/gh/TechnologyAdvice/stardust/1315